src/utils/find.ts

```diff
@@ -42,7 +42,7 @@ function matches(node: VNode, selector: FindAllComponentsSelector): boolean {
 
     // if a name is missing, then check the locally registered components in the parent
     if (node.component.parent) {
-      const registry = (node.component.parent as any).components
+      const registry = (node.component.parent as any).type.components
```

Comment:
Nice, do we need a check to see if `type` is not null?

Reply:
Good question but it is not marked as nullable in the TS definition of `ComponentInternalInstance`, so I think we're good (but I do agree we would feel safer if we had `strict: true`)